src/locale/pages/organizeJourneyInstance/en.yml

```diff
@@ -9,6 +10,7 @@ editButton: Edit
 expandButton: Expand
 members: Members
 milestones: Milestones
+noMilestonesMessage: There are no milestones.
```

Comment:
I think `Message` is redundant, since in some way everything in these files are messages.